clusterloader2/testing/experimental/storage/pod-startup/volume-types/persistentvolume/pvc.yaml

```diff
@@ -5,11 +5,13 @@ metadata:
   labels:
     app: {{.Name}}
     group: {{.Group}}
+  annotations:
+    volume.beta.kubernetes.io/storage-provisioner: {{.Provisioner}}
```

Comment:
Gotcha :sob: I assumed that the actual value would still get added, but I didn't test that. Now I tested and the volume provisioning gets stuck because it remains empty. I'll revise this.